java-frontend/src/test/files/se/ConditionAlwaysTrueOrFalseCheck.java

```diff
@@ -2123,3 +2123,14 @@ void bar(A a) {
     }
   }
 }
+
+class OptionalEmptyNotPresent {
+  private Optional fun() {
+    java.util.Optional<String> op = java.util.Optional.empty();
+    if (op.isPresent()) { // Noncompliant
```

Comment:
I would add the message of the issue, to confirm it's an "always false", and not an "always true"